src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Analyzers/CustomTypeMarshallerAnalyzer.cs

```diff
@@ -345,7 +345,7 @@ public static class MissingMemberNames
         public static readonly DiagnosticDescriptor LinearCollectionElementTypesMustMatchRule =
             new DiagnosticDescriptor(
                 Ids.InvalidSignaturesInMarshallerShape,
-                "InvalidSignaturesInMarshallerShape",
```

Comment:
Just curious do we consistently use Marshaller spelling? As opposed to Marshaler.

Reply:
As of two-ish years ago, yes. Our new APIs / code should be consistently using the double `l`. We settled on: marshal, marshalling, marshalled, marshaller. We've also been working with docs style guide owners to try to get an entry added.